example_configs/tomcat.json

```diff
@@ -0,0 +1,34 @@
+{
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>requestCount:",
+      "name": "tomcat_request_count",
```

Comment:
Do you have an example /metrics that this produces?

This should probably be tomcat_requests_total